### PR TITLE
[IMP] *: black lints

### PR DESCRIPTION
**Pull request by ajqn9094**
Black lints corrected on all modules

### CODE REVIEW

**Comment by ajqn9094** [comment]
@luisg123v Could you review please

Thanks in advance

**Review by luisg123v — CHANGES_REQUESTED** [review]
`[IMP] module_name: black, isort, prettier`

Keep that order and don't include task ID

**Comment by ajqn9094** [comment]
> `[IMP] module_name: black, isort, prettier`
> 
> Keep that order and don't include task ID

Done

File: account_analytic_business_unit/__manifest__.py [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
-        'analytic'
+    "name": "Analytic Business Units",
+    "version": "15.0.1.0.0",
+    "category": "Accounting/Accounting",
+    "summary": """If you set a business unit in you analytic account now you can have an easier way to group by that
+    concept on all your analytic accounting""",
+    "author": "Vauxoo",
+    "website": "https://vauxoo.com",
+    "license": "LGPL-3",
+    "depends": ["account", 'analytic'],
```
Comment (luisg123v) [inline account_analytic_business_unit/__manifest__.py]: - Double quotes
- Add trailing comman so a module per line si preserved

File: stock_by_warehouse_mrp_bom/__manifest__.py [hunk 2]
```diff
@@ -13,12 +13,8 @@
         "mrp",
         "stock_by_warehouse",
     ],
-    "data": [
-        'views/mrp_bom_line_views.xml',
-        'views/mrp_bom_views.xml'
-    ],
-    "demo": [
-    ],
+    "data": ["views/mrp_bom_line_views.xml", "views/mrp_bom_views.xml"],
```
Comment (luisg123v) [inline stock_by_warehouse_mrp_bom/__manifest__.py:16]: Trailing comma

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: account_analytic_business_unit/__manifest__.py (hunk 1 above)
Comment (ajqn9094) [reply account_analytic_business_unit/__manifest__.py]: Done

File: stock_by_warehouse_mrp_bom/__manifest__.py (hunk 2 above)
Comment (ajqn9094) [reply stock_by_warehouse_mrp_bom/__manifest__.py:16]: Done

**Comment by ajqn9094** [comment]
@luisg123v Changes applied requested applied, additionally similar details found in the other commits are corrected.

File: default_warehouse_from_sale_team/models/crm_team.py [hunk 3]
```diff
@@ -5,27 +5,25 @@ class CrmTeam(models.Model):
     _inherit = "crm.team"
 
     default_warehouse_id = fields.Many2one(
-        'stock.warehouse',
-        help='In this field can be defined a default warehouse for '
-        'the related users to the sales team.',
+        "stock.warehouse",
+        help="In this field can be defined a default warehouse for " "the related users to the sales team.",
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/models/crm_team.py]:
```suggestion
        help="In this field can be defined a default warehouse for the related users to the sales team.",
```

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: default_warehouse_from_sale_team/models/crm_team.py (hunk 3 above)
Comment (ajqn9094) [reply default_warehouse_from_sale_team/models/crm_team.py]: Done

**Review by luisg123v — APPROVED** [review]

**Pull request merged**